4_advanced_usage/77_distributed_rudder.txt

```diff
@@ -73,6 +73,106 @@ cd rudder-multiserver-setup
 
 ----
 
+==== Install rudder-db manually
+
+The setup is done in two places: on the database server and on the Rudder root server.
+It also allows moving an existing database to another server.
+
+===== On the database server
+
+* Install and configure the agent on the node
+
+* Install the *rudder-reports* package
+
+* Change postgresql configuration (postgresql.conf) to listen on the right interface to communicate with the server:
```

Comment:
Could you provide a rough idea of the path to this file? If it varies, you could give examples, like "usually in /etc or /etc/pgsql/"